hw/system/occamy/src/occamy_cfg.hjson

```diff
@@ -168,10 +154,102 @@
         num_itlb_entries: 1,
     }
     // peripherals
+    debug: {
+        address: 0, // 0x0000_0000
+        length: 4096, // 4 kiB 0x1000
+    },
+    rom: {
+        address: 16777216, // 0x0100_0000
+        length: 131072, // 128 kiB 0x2_0000
+    },
+    soc_ctrl: {
+        address: 33554432, // 0x0200_0000
+        length: 4096, // 4 kiB 0x1000
+    },
+    clk_mgr: {
+        address: 33558528, // 0x0200_1000
+        length: 4096, // 4 kiB 0x1000
+    },
+    uart: {
+        address: 33562624, // 0x0200_2000
+        length: 4096, // 4 kiB 0x1000
+    },
+    gpio: {
+        address:  33566720, // 0x0200_3000
+        length: 4096, // 4 kiB 0x1000
+    },
+    i2c: {
+        address: 33570816, // 0x0200_4000
+        length: 4096, // 4 kiB 0x1000
+    },
+    chip_ctrl: {
+        address: 33574912, // 0x0200_5000
+        length: 4096, // 4 kiB 0x1000
+    },
+    timer: {
+        address: 33579008, // 0x0200_6000
+        length: 4096, // 4 kiB 0x1000
+    },
+    spim: {
+        address: 50331648, // 0x0300_0000
+        length: 131072, // 4 kiB 0x2_0000
+    },
     clint: {
-        // 0x0400_0000
-        address: 67108864,
-        // 0x0010_0000
-        length: 1048576
+        address: 67108864, // 0x0400_0000
+        length: 1048576, // 1 MiB 0x10_0000
+    },
+    pcie_cfg: {
+        address: 83886080, // 0x0500_0000
+        length: 131072, // 128 kiB 0x2_0000
+    },
+    hbi_cfg: {
+        address: 100663296, // 0x0600_0000
+        length: 65536, // 64 kiB 0x1_0000
+    },
+    hbi_ctl: {
+        address: 117440512, // 0x0700_0000
+        length: 65536, // 64 kiB 0x1_0000
+    },
+    hbm_cfg: {
+        address: 134217728, // 0x0800_0000
+        length: 4194304, // 4 MiB 0x40_0000
+    },
+    hbm_phy_cfg: {
+        address: 150994944 // 0x0900_0000
+        length: 1048576, // 1 MiB 0x10_0000
+    },
+    hbm_seq: {
+        address: 167772160, // 0x0A00_0000
+        length: 65536, // 64 kiB 0x1_0000
+    },
+    plic: {
+        address: 201326592, // 0x0C00_0000
+        length: 67108864, // 64 MiB 0x400_0000
+    },
+    // non-peripheral IPs
+    pcie: {
+      address_io: 536870912, // 0x2000_0000
+      address_mm: 1207959552, // 0x4800_0000
+      length: 671088640, // 640 MiB 0x2800_0000
+    },
+    spm: {
+      address: 1879048192, // 0x7000_0000
+      length: 131072, // 128 kiB 0x2_0000
+    },
+    hbi: {
+        address: 1099511627776, // 0x100_0000_0000
+        length: 1099511627776, // 1 TiB 0x100_0000_0000
+    }
+    hbm: {
+        address_0: 2147483648, // 0x8000_0000
+        address_1: 68719476736, // 0x10_0000_0000
+        channel_size: 1073741824, // 1 GiB 0x4000_0000
+        nr_channels_total: 8,
+        nr_channels_address_0: 2,
+    },
+    // dram corresponds to 'hbm address_0' and 'nr_channels_address_0'
+    dram: { 
+        address: 2147483648, // 0x8000_0000
+        length: 2147483648, // 2 GiB 0x8000_0000
```

Comment:
I am not sure if we should merge these configs somehow as the `dram` corresponds to `address_0` of `hbm` with `nr_channels_address_0` channels that are mapped into that memory region.